### ISSUE TITLE
Question: Disable webhook for Github provider

### ISSUE THREAD

**Issue by engrun** [awslabs/aws-deployment-framework#337]
We have a case where we need to create pipelines that are not triggered automatically by new commits to GH-repo. 

Looking at the docs, there doesn't seem to an property that disables the webhook creation or in any other way disables triggering of pipelines

Does anybody know it this is at all possible?

Looking at possible workarounds. 
1. Remove webhooks from GH, but I guess these will be re-applied when the "pipelines-pipeline" is re-run?
2. Write a script the de-registers the webhook from the pipeline. definitely will be re-applied when the "pipelines-pipeline" is re-run

3. Create a job that runs at periodic intervals that does either 1 og 2

Thoughts from the community?

**Comment by thomasmcgannon** [comment]
Hi Rune,
In your use case, how would you expect the pipeline to be triggered, would this be through manually running the pipeline?
Currently there is no way to stop a pipeline from executing when there is a change in the source, however a similar feature (#308) was also requested.

Would having an option in your pipeline definition to not execute automatically solve your problem?

**Comment by benbridts** [comment]
I haven't tested this, but you could solve part of this by adding an approval before the first deploy (the build would still run though)

**Comment by engrun** [comment]
@thomasmcgannon 
thanks for the pointer to the other issue. This is in fact one of our cases. The other being manually triggering the pipeline as you say. 

And yes, and option for this in the yaml would be perfect. 👍 
eg. 
```
Enabled : True | False
or 
Trigger_on_update : True | False
```

**Comment by sbkok** [comment]
Thank you for your patience. I am happy to inform you that this feature has been [released as part of v3.2.0](https://github.com/awslabs/aws-deployment-framework/releases/tag/v3.2.0) just now.
I'm hereby closing this issue. Please open a new issue if you are experiencing any issues related to this feature.
